### PR TITLE
Add RH col with ad to crossword at wide breakpoint

### PR DESCRIPTION
**Pull request by adamnfish**
Need to clear this change with Comms prior to merge / release, because crosswords.

## What does this change?

If ads are enabled we add a secondary-column to the crossword page that only appears at the largest breakpoint. It contains an ad slot that can fit ads up to 300px wide.

The crossword layout has been tweaked to accommodate the ad in at this breakpoint, based on the design in the ticket. The design given wasn't fully realisable so only this breakpoint works like this, happy to chat about this more in person. We can look at designing something for smaller breakpoints if this works.

## Screenshots

crossword-ad

## What is the value of this and can you measure success?

People persist on the crossword page so a refreshing ad there (even only at the largest size) could provide real value from our free online crosswords.

### Does this affect other platforms?

It does not.

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

No, this is just a small tweak to the crossword page.

### Does this change break ad-free?

- [x] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Accessibility test checklist

No change to how the crosswords work here.

### Tested

- [x] Locally
- [x] On CODE (optional)

Looks ok at all breakpoints, tested with multiple crosswords of all types. Crosswords still work, too.

### CODE REVIEW

File: static/src/stylesheets/module/crosswords/_clues.scss [hunk 1]
```diff
@@ -14,7 +14,7 @@
 .crossword__clue {
     @include fs-textSans(4);
     position: relative;
-    padding: $gs-baseline * .25 $gs-baseline;
+    padding: $gs-baseline * .25 $gs-baseline $gs-baseline * .25 0;
```
Comment (adamnfish) [inline static/src/stylesheets/module/crosswords/_clues.scss]: The new design has clues left-aligned with the heading, this removes left padding.

File: static/src/stylesheets/module/crosswords/_layout.scss [hunk 2]
```diff
@@ -51,6 +51,10 @@
     clear: both;
     position: relative;
     margin-top: $gs-baseline;
+
+    @include mq($from: wide) {
+        margin-right: gs-span(4) + $gs-baseline;
+    }
```
Comment (adamnfish) [inline static/src/stylesheets/module/crosswords/_layout.scss]: Add space for the secondary-column at the wide breakpoint, where it appears.

File: static/src/stylesheets/module/crosswords/_clues.scss [hunk 3]
```diff
@@ -42,10 +42,6 @@
 
     .has-grouped-clues & {
         width: $gs-gutter * 2;
-
-        @include mq(leftCol) {
-            width: $gs-gutter * 3;
-        }
```
Comment (adamnfish) [inline static/src/stylesheets/module/crosswords/_clues.scss]: The new design calls for tighter clue boxes to fit the ad column. Removes the extra space at the leftCol breakpoint as well so it's consistent at larger sizes.

File: applications/app/views/fragments/crosswords/crosswordContent.scala.html [hunk 4]
```diff
@@ -36,6 +38,11 @@ <h3 class="crossword__clues-header">Down</h3>
                     </div>
                 </div>
             </div>
+            @if(CommercialSwitch.isSwitchedOn && shouldShowAds(crosswordPage)) {
```
Comment (kelvin-chappell) [inline applications/app/views/fragments/crosswords/crosswordContent.scala.html]: I wonder whether `shouldShowAds` should hold the switch state, so that it doesn't get forgotten.  But not really part of this PR

File: applications/app/views/fragments/crosswords/crosswordContent.scala.html [hunk 5]
```diff
@@ -36,6 +38,11 @@ <h3 class="crossword__clues-header">Down</h3>
...
+                <div class="content__secondary-column hide-until-wide" aria-hidden="true">
+                @fragments.commercial.standardAd("right", Seq("mpu-banner-ad"), Map("desktop" -> Seq("300,250", "300,274", "300,600", "fluid")))
```
Comment (kelvin-chappell) [inline applications/app/views/fragments/crosswords/crosswordContent.scala.html]: Is 300x274 a valid size?

Comment (ScottPainterGNM) [reply applications/app/views/fragments/crosswords/crosswordContent.scala.html]: It's the size for the google backfill (ad label funzies)

Comment (adamnfish) [reply applications/app/views/fragments/crosswords/crosswordContent.scala.html]: Good question. I didn't make it up though!

https://github.com/guardian/frontend/search?q=300%2C274&unscoped_q=300%2C274

Comment (adamnfish) [reply applications/app/views/fragments/crosswords/crosswordContent.scala.html]: Ah, thanks @ScottPainterGNM.

**Review by kelvin-chappell — APPROVED** [review]
How refreshing! 💯

**Comment by ScottPainterGNM** [comment]
👍 comms are done and both design and editorial have signed this off
